ui/components/component-library/help-text/help-text.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import classnames from 'classnames';
+import {
+  TextVariant,
+  TextColor,
+} from '../../../helpers/constants/design-system';
+import { Text, ValidTag } from '../text';
+import { HelpTextProps, HelpTextSeverity } from './help-text.types';
+
+export const HelpText = ({
```

Comment:
I know this wasn't in the original issue but could we please forwardRefs for this component. I think it makes sense to for all of our components